ci/unit/test_post_always_node.sh

```diff
@@ -3,15 +3,27 @@
 set -uex
 
 cd "$DAOS_BASE"
-mkdir run_test.sh
-mkdir vm_test
-mv nlt-errors.json vm_test/
-if ls /tmp/daos*.log > /dev/null; then
-  mv /tmp/daos*.log run_test.sh/
-fi
-if ls /tmp/dnt*.log > /dev/null; then
-  mv /tmp/dnt*.log vm_test/
+
+if [ "$WITH_VALGRIND" = "memcheck" ]; then
+    mkdir run_test_memcheck.sh
```

Comment:
Does this directory need to be named differently here or can it be the same name for all variants of this stage and the renaming done at the end just prior to adding it as an artifact?  It would simplify and DRY this code out to use the same name intermediately and defer the rename until the end of the stage, just prior to adding artifacts.

Reply:
do you mean do not rename on test_post_always_node.sh and do it in unitTestPost.groovy ?
just before this https://github.com/daos-stack/pipeline-lib/blob/master/vars/unitTestPost.groovy#L50
I don't know if it needs to be changed here, but I can send a patch with the changes and see how it works. Thanks for the feedback

Reply:
Perhaps like that.  But the idea is to minimise the valgrind/no-valgrind branching and code duplication by deferring the renaming of the artifacts into valgrind/no-valgrind names at the very end and let all of the processing before just use the same name in both paths.